aspnetcore/mvc/views/tag-helpers/th-components/samples/RazorPagesSample/Pages/Shared/_Layout.cshtml

```diff
@@ -0,0 +1,73 @@
+﻿<!DOCTYPE html>
+<html>
```

Comment:
I think they are planning to `lang="en"` these `<html>` tags btw for either 2.2 or 3.0 templates. You don't have to change it tho ... no biggie. It's something that I've been doing on all of our sample apps.